src/library_wasmfs.js

```diff
@@ -157,7 +157,18 @@ FS.createPreloadedFile = FS_createPreloadedFile;
       });
     },
     // TODO: lchmod
+    lchmod: (path, mode) => {
+      return withStackSave(() => {
+        var buffer = stringToUTF8OnStack(path);
+        return __wasmfs_lchmod(buffer, mode);
+      })
+    },
     // TODO: fchmod
+    fchmod: (fd, mode) => {
+      return withStackSave(() => {
```

Comment:
Got it. Just for my own understanding, we don't need `withStackSave` here because we don't have to convert a string to `UTF8Array` like we had to do with the file path in `chmod`, right?

Reply:
Yes, exactly. `stringToUTF8OnStack` saves the string data on the stack, so we need to save&restore the stack using `withStackSave`. Without any stack usage, like in `fchmod`, that is not needed.